src/app/api/stacks/clients/hiro-api-client.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+import { type DefaultNetworkConfigurations, defaultNetworksKeyedById } from '@shared/constants';
+
+export const leatherHeaders = {
+  'x-leather-version': VERSION,
+  'x-hiro-product': 'leather',
+};
+
+export function hiroApiClient(network: DefaultNetworkConfigurations) {
+  return axios.create({
+    baseURL: defaultNetworksKeyedById[network].chain.stacks.url,
+    headers: { ...leatherHeaders },
+  });
```

Comment:
Mmm, actually (contrary to my earlier recommendation), this wouldn't work with custom networks, should just pass the URL

Reply:
I see this now, thx.